cli/testdata/coder_create_--help.golden

```diff
@@ -14,6 +14,9 @@ OPTIONS:
           Specify automatic updates setting for the workspace (accepts 'always'
           or 'never').
 
+      --copy-parameters string, $CODER_WORKSPACE_COPY_PARAMETERS
```

Comment:
Nit: `--copy-parameters-from`? Up to you.

Reply:
Done